tests/slack/slack.go

```diff
@@ -11,8 +11,9 @@ import (
 )
 
 var (
-	Channel    string
-	WebhookURL string
+	Channel      string
+	WebhookURL   string
+	SuccessCount int
```

Comment:
`SuccessCount` is not used?

Reply:
It is used at these postions:

https://github.com/pingcap/tidb-operator/pull/579/files#diff-2ef72aa70c90226cd8aac8a436f9bff9R150
https://github.com/pingcap/tidb-operator/pull/579/files#diff-8f8eafaeec65b7926966885b1c103ca8R58
https://github.com/pingcap/tidb-operator/pull/579/files#diff-8f8eafaeec65b7926966885b1c103ca8R300